packages/legend-query/src/stores/__tests__/TEST_DATA__QueryBuilder_Model_M2MWithInheritance.json

```diff
@@ -0,0 +1,749 @@
+[
+  {
+    "path": "model::target::IncType",
+    "content": {
+      "_type": "Enumeration",
+      "name": "IncType",
+      "package": "model::target",
+      "values": [
+        {
+          "value": "LLC"
+        },
+        {
+          "value": "CORP"
+        }
+      ]
+    },
+    "classifierPath": "meta::pure::metamodel::type::Enumeration"
+  },
+  {
+    "path": "model::target::NFirm",
+    "content": {
+      "_type": "class",
+      "constraints": [
+        {
+          "functionDefinition": {
+            "_type": "lambda",
+            "body": [
+              {
+                "_type": "func",
+                "function": "startsWith",
+                "parameters": [
+                  {
+                    "_type": "property",
+                    "parameters": [
+                      {
+                        "_type": "var",
+                        "name": "this"
+                      }
+                    ],
+                    "property": "name"
+                  },
+                  {
+                    "_type": "string",
+                    "multiplicity": {
+                      "lowerBound": 1,
+                      "upperBound": 1
+                    },
+                    "values": ["MC"]
+                  }
+                ]
+              }
+            ],
+            "parameters": []
+          },
+          "name": "namePrefix"
+        }
+      ],
+      "name": "NFirm",
+      "package": "model::target",
+      "properties": [
+        {
+          "multiplicity": {
+            "lowerBound": 1
+          },
+          "name": "nEmployees",
+          "type": "model::target::NPerson"
+        },
+        {
+          "multiplicity": {
+            "lowerBound": 1,
+            "upperBound": 1
+          },
+          "name": "name",
+          "type": "String"
+        },
+        {
+          "multiplicity": {
+            "lowerBound": 1,
+            "upperBound": 1
+          },
+          "name": "incType",
+          "type": "model::target::IncType"
+        }
+      ],
+      "qualifiedProperties": [
+        {
+          "body": [
+            {
+              "_type": "func",
+              "function": "first",
+              "parameters": [
+                {
+                  "_type": "property",
+                  "parameters": [
+                    {
+                      "_type": "var",
+                      "name": "this",
+                      "sourceInformation": {
+                        "endColumn": 24,
```

Comment:
@MauricioUyaguari do we generally add `sourceInformation` in the roundtriptests we add in `studio`?

Reply:
@YannanGao-gs  please remove source information. 
Easy way to do this. is load through model loader. Save workspace -> reload app -> model loader again and this time load entities from project. That will not have sourceInformation.